### ISSUE TITLE
Dependency not uploaded?

### ISSUE THREAD

**Issue by eric239** [t9t/jooq-postgresql-json#1]
Hi, trying to use your repo (in gradle, `implementation 'com.github.t9t.jooq:jooq-postgresql-json:0.2.0'`) getting
```
  > Could not resolve com.github.t9t.jooq:jooq-postgresql-json:0.2.0.
     Required by:
         project :
      > Could not resolve com.github.t9t.jooq:jooq-postgresql-json:0.2.0.
         > Could not parse POM https://jcenter.bintray.com/com/github/t9t/jooq/jooq-postgresql-json/0.2.0/jooq-postgresql-json-0.2.0.pom
            > Could not find com.github.t9t.jooq:jooq-postgresql-json-parent:0.2.0.
```
Seems like the parent POM is not uploaded to maven central

**Comment by bfg** [comment]
Applies to version `0.3.0` as well; parent pom is not deployed to maven central, see:

https://javalibs.com/artifact/com.github.t9t.jooq/jooq-postgresql-json-parent

I had to add jar as a file dependency :/

```gradle
dependencies {
  // other dependencies

  implementation files("${rootProject.rootDir}/lib/jooq-postgresql-json-0.3.0.jar")
}
```

**Comment by t9t** [comment]
Hi @eric239 and @bfg, thanks for reporting this issue. I'm extremely sorry for not publishing the artifact correctly and also not seeing this issue!

I had no experience yet pushing a multi-module project to OSSRH, so while trying to exclude the integration test module, I also accidentally excluded the parent project! I didn't notice this myself, because locally everything worked fine: the artifacts were installed to my local repository.

On top of that, I was apparently not watching my own project, so Github was not sending me any emails for issues that were created... I only noticed when I got explicitly tagged on another issue.

I was unable to restore version 0.3.0, so I have released it again as 0.3.1. I have also tested this locally by first clearing the artifacts from my local repository, so all should be fine now. Let me know if it works or not.

```
<dependency>
  <groupId>com.github.t9t.jooq</groupId>
  <artifactId>jooq-postgresql-json</artifactId>
  <version>0.3.1</version>
</dependency>
```
